### PR TITLE
Wishbone connected to pins for synthesis benchmark

### LINKED ISSUE #242: Connect only Wishbone pins in the synthesis script
We now have a LSU unit, so we might just connect both Wishbone buses to I/O pins for correct synthesis benchmarking.

### PR DESCRIPTION
**Pull request by tilk**
Fixes #242.

### CODE REVIEW

File: constants/ecp5_platforms.py [hunk 1]
```diff
@@ -16,15 +46,23 @@ class ECP5BG381Platform(LatticeECP5Platform):
...
+            dat_w=p(32),
+            rst=p(),
+            ack=p(),
+            adr=p(30),
+            cyc=p(),
+            stall=p(),
+            err=p(),
+            lock=p(),
+            rty=p(),
+            sel=p(4),
```
Comment (Kristopher38) [inline constants/ecp5_platforms.py:63]: why is sel width hardcoded?

File: constants/ecp5_platforms.py [hunk 2]
```diff
@@ -16,15 +46,23 @@ class ECP5BG381Platform(LatticeECP5Platform):
     default_rst = "rst"
 
     resources = [
-        Resource("rst", 0, PinsN(ecp5_bg381_pins[0], dir="i"), Attrs(IO_TYPE="LVCMOS33")),
-        Resource("clk", 0, Pins(ecp5_bg381_pins[1], dir="i"), Clock(12e6), Attrs(IO_TYPE="LVCMOS33")),
-        Resource(
-            "data_out",
+        Resource("rst", 0, PinsN(p(), dir="i"), Attrs(IO_TYPE="LVCMOS33")),
+        Resource("clk", 0, Pins(p(), dir="i"), Clock(12e6), Attrs(IO_TYPE="LVCMOS33")),
```
[the remark this answers is not in the document]
Comment (tilk) [reply constants/ecp5_platforms.py:50]: In the context of synthesis, probably not. Does it change the performance values? I don't know.

File: constants/ecp5_platforms.py (hunk 1 above)
Comment (tilk) [reply constants/ecp5_platforms.py:63]: Same reason why the dat_r, dat_w and adr widths are hardcoded: the platform is not parametrized. Maybe it should be?

File: scripts/synthesize [hunk 3]
```diff
@@ -5,79 +5,66 @@ import sys
...
-        from coreblocks.transactions import TransactionModule
-        from coreblocks.peripherals.wishbone import WishboneMaster, WishboneMemorySlave, WishboneParameters
 
         m = Module()
         tm = TransactionModule(m)
-        wb_params = WishboneParameters(data_width=32, addr_width=30)
 
-        self.wb_master = WishboneMaster(wb_params=wb_params)
-        self.wb_mem_slave = WishboneMemorySlave(wb_params=wb_params, width=32, depth=32, init=self.instr_mem)
+        wb_params = WishboneParameters(data_width=32, addr_width=30)
```
[the remark this answers is not in the document]
Comment (tilk) [reply scripts/synthesize:33]: Out of scope, I think this naturally fits into what @xThaid wants to do.

File: constants/ecp5_platforms.py (hunk 1 above)
Comment (tilk) [reply constants/ecp5_platforms.py]: Changed.

File: constants/ecp5_platforms.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tilk) [reply constants/ecp5_platforms.py]: Connected clock to a primary clock pin on the chip.

File: scripts/synthesize (hunk 3 above)
[the remark this answers is not in the document]
Comment (xThaid) [reply scripts/synthesize]: Yes, I also thought of doing that. So please leave it as it is and I will change it in my PR.

**Review by lekcyjna123 — APPROVED** [review]

File: scripts/synthesize.py [hunk 4]
```diff
@@ -5,78 +5,67 @@
...
         m.submodules.c = self.core
 
         return tm
 
 
 def synthesize(platform: str):
-    from coreblocks.params.genparams import GenParams
-    from constants.ecp5_platforms import ECP5BG381Platform
-    from coreblocks.params.configurations import basic_configuration
+    from constants.ecp5_platforms import make_ecp5_platform
```
Comment (xThaid) [inline scripts/synthesize.py:59]: NIT: this import can be also moved out of this function.

**Review by xThaid — APPROVED** [review]

**Pull request merged**